packages/components/package.json

```
@@ -31,6 +31,7 @@
 	"dependencies": {
 		"classnames": "^2.2.6",
 		"enzyme": "^3.10.0",
+		"flag-icon-css": "3.4.5",
```

this should be a caret rev 
```suggestion
		"flag-icon-css": "^3.4.5",
```